packages/bpk-docs/src/routes/Routes.js

```diff
@@ -97,15 +98,19 @@ const Routes = (
 
     <Route path={ROUTES.STYLE} component={StylePage} iconsSvgs={iconsSvgs} />
 
-    <Route path={ROUTES.COMPONENTS} component={DocsLayout}>
-      <IndexRedirect to={ROUTES.WEB_COMPONENTS} />
+    <Route path={ROUTES.TOKENS} component={TokensLayout}>
+      <IndexRedirect to={ROUTES.ANIMATION} />
+      <Route path={ROUTES.ANIMATION} component={AnimationPage} />
+      <Route path={ROUTES.BORDERS} component={BordersPage} />
       <Route path={ROUTES.COLORS} component={ColorsPage} />
-      <Route path={ROUTES.TYPESETTING} component={TypesettingPage} />
+      <Route path={ROUTES.LAYOUT} component={LayoutPage} />
       <Route path={ROUTES.RADII} component={RadiiPage} />
       <Route path={ROUTES.SHADOWS} component={ShadowsPage} />
-      <Route path={ROUTES.BORDERS} component={BordersPage} />
-      <Route path={ROUTES.LAYOUT} component={LayoutPage} />
-      <Route path={ROUTES.ANIMATION} component={AnimationPage} />
+      <Route path={ROUTES.TYPESETTING} component={TypesettingPage} />
+    </Route>
+
```

Comment:
The additional whitespace here is inconsistent with the rest of the file :)

Reply:
Actually, each top level route (i.e. using, style, tokens, components & resources) has new lines around it.

Reply:
My bad!

Reply:
Not at all - the diff doesn't do it justice!